btcclient/client_wallet.go

```diff
@@ -35,7 +35,7 @@ func NewWallet(cfg *config.BTCConfig, parentLogger *zap.Logger) (*Client, error)
 		HTTPPostMode: true,
 		User:         cfg.Username,
 		Pass:         cfg.Password,
-		DisableTLS:   cfg.DisableClientTLS,
```

Comment:
why remove from config?

Reply:
wait this was confusing

Reply:
not sure why this appeared to me as code change at first

Reply:
This was in previous PR, basically we are not using it in bitcoind.